### PR TITLE
Add tile component

### PR DESCRIPTION
**Pull request by tarikhamilton**
Check it out. It will 99% have some future updates, so I'd look for smell over anything.

Basic transitions. We can add the cool animated border in an update.

https://app.zeplin.io/project/5b4e44c4113211861caaa7c1/screen/5c1bf894420c9e112ae060d5

### CODE REVIEW

File: components/Tile/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,75 @@
+@import "styles/index";
+
+$shadow: 0 2px 22px 0 rgba(0, 0, 0, 0.15);
+
+.tile {
+  position: relative;
+  overflow: hidden;
+  display: inline-flex;
+  flex-direction: column;
+  background-color: white;
```
Comment (jdashley27) [inline components/Tile/index.module.scss]: Overall, this looks great. A minor question I have is, do we want to tie in this white to the __colors '$color-white' variable?

File: components/Tile/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,75 @@
+@import "styles/index";
+
+$shadow: 0 2px 22px 0 rgba(0, 0, 0, 0.15);
```
Comment (jdashley27) [inline components/Tile/index.module.scss]: Would this work being moved into the colors file, so other parts of MDL can make use of it?

File: components/Tile/index.module.scss [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+.tile {
+  position: relative;
+  overflow: hidden;
+  display: inline-flex;
+  flex-direction: column;
+  background-color: white;
+  width: 220px;
```
Comment (josephfusco) [inline components/Tile/index.module.scss]: If this is the same tile/card that we are using in our WP sites, having a fixed width of 220px may be too opinionated for this component. IMO these types of components should be fluid and fill their parent layout component. We can still add a max-width though.

**Comment by josephfusco** [comment]
@tarikhamilton nice work on this! 🔥

**Pull request merged**